source/find_getmore_killcursors_commands.rst

```diff
@@ -447,7 +444,7 @@ Semantics of maxTimeMS for a Driver
 
 In the case of  a **non-tailable cursor query** OR **a tailable cursor query with awaitData == false**, the driver MUST set maxTimeMS on the **find** command and MUST NOT set maxTimeMS on the **getMore** command.
 
-In the case of **a tailable cursor with awaitData == true**, the driver MUST set maxTimeMS on both the** find** and subsequent **getMore** commands.
+In the case of **a tailable cursor with awaitData == true**, the driver MAY set the maxTimeMS field on both the **find** and subsequent **getMore** commands. In the case where the driver allows maxTimeMS to be set on the **getMore** command it SHOULD be exposed in the CRUD API as the option named **maxAwaitTimeMS**.
```

Comment:
I'll get this into the CRUD spec. However, this change in here hasn't provided any guidance. You need to be very clear about whether or not to set maxTimeMS on getMore or not. This should be a MUST or a MUST NOT. MAY simply means every driver will pick something different. In this case, since the previous sentence states what to do with non-tailable or tailable and no await data, I'd just remove the qualification and say this:

The driver MUST set maxTimeMS on the **find** command and MUST NOT set maxTimeMS on the **getMore** command. In the case of a tailable cursor with awaitData set, the driver MUST set maxTimeMS on the **getMore** to the maxAwaitTimeMS setting in the driver.